identity-vault/getting-started/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "getting-started",
```

Comment:
We should name this something more specific. The Auth Connect Getting Started guide uses `ac-getting-started`, so `iv-getting-started` would be appropriate here.

Reply:
FWIW, the reason we do this is so the names are unique for the `--filter`

So we can do `pnpm --filter iv-getting-started build` or whatever to build this. If there were multiple packages with the same name we couldn't do this and would have to resort to using a full path for the filter which is more clumsy.